src/components/Form/Form.tsx

```diff
@@ -25,14 +25,17 @@ export const Form: FC<FormProps> = ({ activity }) => {
   const { loading, form, error, refetch } = useForm(activity)
   const { t } = useTranslation()
   const [evaluateFormRules] = useEvaluateFormRules(activity.object.id)
-  const { onSubmit } = useSubmitForm({ activity })
+  const { onSubmit, isSubmitting } = useSubmitForm({ activity })
   const { branding } = useHostedSession()
 
   const [formProgress, setFormProgress] = useLocalStorage(activity.id, '')
 
   if (loading) {
     return <LoadingPage title={t('activities.form.loading')} />
   }
+  if (isSubmitting) {
+    return <LoadingPage title={t('activities.form.submitting')} />
+  }
```

Comment:
I understand why you added a loader here.

Wouldn't this cause "_blinking_" on the screen?

User clicks on form submit button
-> Page abruptly changes to loading
Activity completed, now waiting for next activity
-> Page abruptly changes to loading again
Form Activity loaded
Fetching form
-> Page abruptly changes to loading again
Form fetched


I would've expected a loader on the "Submit" button (disable button, show a spinner) than showing a loader on the full page. Wdyt?